### PR TITLE
feature(common): the whole process for placing an order is added [WEB…

### PR DESCRIPTION
**Pull request by juan865010**
the whole process for placing an order is added

### CODE REVIEW

**Comment by olmedo90** [comment]
image

image

image

image

It is not according to Figma. correct the following: (size of the texts, space between the inputs name and surname, add checkbox lorem, modify the size of the checkbox shipping delivery payment)

**Comment by olmedo90** [comment]
image
image

It is according to Figma. continue with the same interest.👍

File: src/ui/components/confirmOrder/ConfirmOrder.jsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import useStyles from "./confirmOrder.styles";
```
Comment (Ditmar) [inline src/ui/components/confirmOrder/ConfirmOrder.jsx]: please use single quotes

File: src/ui/components/confirmOrder/ConfirmOrder.jsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+import OrderSend from "../infoFacSend/orderSend/OrderSend";
```
Comment (Ditmar) [inline src/ui/components/confirmOrder/ConfirmOrder.jsx]: please use single quotes

File: src/ui/components/confirmOrder/ConfirmOrder.jsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import Confirm from "./confirm/Confirm";
```
Comment (Ditmar) [inline src/ui/components/confirmOrder/ConfirmOrder.jsx]: please use single quotes

File: src/ui/components/confirmOrder/confirm/Confirm.jsx [hunk 4]
```diff
@@ -0,0 +1,83 @@
+import useStyles from "./confirm.styles";
```
Comment (Ditmar) [inline src/ui/components/confirmOrder/confirm/Confirm.jsx]: please use single quotes

File: src/ui/components/infoFacSend/InfoFacSend.jsx [hunk 5]
```diff
@@ -0,0 +1,19 @@
+import useStyles from "./InfoFacSend.styles";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/InfoFacSend.jsx]: please use single quotes

File: src/ui/components/infoFacSend/InfoFacSend.jsx [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+import OrderSend from "./orderSend/OrderSend";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/InfoFacSend.jsx]: please use single quotes

File: src/ui/components/infoFacSend/InfoFacSend.jsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+import CheckForm from "./checkForm/CheckForm";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/InfoFacSend.jsx]: please use single quotes

File: src/ui/components/infoFacSend/InfoFacSend.jsx [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+import BreadCrumb from "./breadCrumb/BreadCrumb";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/InfoFacSend.jsx]: please use single quotes

File: src/ui/components/infoFacSend/breadCrumb/BreadCrumb.jsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import Breadcrumbs from "@mui/material/Breadcrumbs";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/breadCrumb/BreadCrumb.jsx]: please use single quotes

File: src/ui/components/infoFacSend/breadCrumb/BreadCrumb.jsx [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+import Link from "@mui/material/Link";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/breadCrumb/BreadCrumb.jsx]: please use single quotes

File: src/ui/components/infoFacSend/breadCrumb/BreadCrumb.jsx [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+import HorizontalRuleIcon from "@mui/icons-material/HorizontalRule";
```
Comment (Ditmar) [inline src/ui/components/infoFacSend/breadCrumb/BreadCrumb.jsx]: please use single quotes

**Review by Ditmar — CHANGES_REQUESTED** [review]
nice pr great work

**Pull request merged**